src/Ledger/Ratify.lagda

```diff
@@ -222,12 +413,42 @@ accepted' Γ es@record { cc = cc , _    ; pparams = pparams , _ }
     acceptedBy : GovRole → Set
     acceptedBy role = let t = threshold pparams (Data.Maybe.map proj₂ cc) action role in
       case totalStake role cc' redStakeDistr votes of λ where
-        0         → t ≡ R.0ℚ -- if there's no stake, accept only if the threshold is zero
-        x@(suc _) → Z.+ acceptedStake role cc' redStakeDistr votes R./ x R.≥ t
+        0          → t ≡ R.0ℚ -- if there's no stake, accept only if the threshold is zero
+        x@(suc _)  → Z.+ acceptedStake role cc' redStakeDistr votes R./ x R.≥ t
 
 expired : Epoch → GovActionState → Set
 expired current record { expiresIn = expiresIn } = expiresIn <ᵉ current
-
+\end{code}
+} %% end small
+\caption{%Ratify iii:
+Yet more types and proofs for the ratification of governance actions}
+\label{fig:defs:ratify-iii}
+\end{figure*}
+The code in Figure~\ref{fig:defs:ratify-iii} defines yet more types required for ratification of a governance action.
+\begin{itemize}
+  \item \AgdaBound{getStakeDist}         %%% of type \AgdaDatatype{GovRole}~\AgdaDatatype{→}~\AgdaDatatype{ℙ}~\AgdaDatatype{VDeleg}~\AgdaDatatype{→}~\AgdaDatatype{StakeDistrs}~\AgdaDatatype{→}~\AgdaDatatype{VDeleg}~\AgdaDatatype{⇀}~\AgdaDatatype{Coin}:
+  is a partial function that maps a delegate to a \AgdaDatatype{Coin} representing the stake distribution.
```

Comment:
`getStakeDist` computes the stake distribution that applies to the given `GovRole`.